modules.d/35network-manager/module-setup.sh

```diff
@@ -30,8 +30,8 @@ install() {
     inst_multiple ip sed grep
 
     inst NetworkManager
-    inst_multiple -o /usr/{lib,libexec}/nm-initrd-generator
-    inst_multiple -o /usr/{lib,libexec}/nm-daemon-helper
+    inst_multiple -o /usr/{lib,libexec}{,/NetworkManager}/nm-initrd-generator
+    inst_multiple -o /usr/{lib,libexec}{,/NetworkManager}/nm-daemon-helper
```

Comment:
@thaller, @lkundrak: are you folks aware that some put these binaries in `/usr/libexec/NetworkManager`? Have you considered making that the upstream default rather than dropping them directly in `/usr/libexec`?